semcore/d3-chart/src/component/ChartLegend/LegendItem/LegendItem.tsx

```diff
@@ -36,10 +36,14 @@ class LegendItemRoot extends Component<
     ),
   });
 
+  getUniqueID() {
+    const { uid, shape } = this.asProps;
+    return shape === 'Checkbox' ? `chart-legend-item-${uid}` : undefined;
```

Comment:
```suggestion
    return `chart-legend-item-${uid}`;
```
let's return some id always and pass of not it to the shape in getShapeProps